test_util/marathon.py

```diff
@@ -9,7 +9,17 @@
 
 import test_util.helpers
 
+DEFAULT_API_BASE = '/marathon/v2'
 TEST_APP_NAME_FMT = '/integration-test-{}'
+REQUIRED_HEADERS = {'Accept': 'application/json, text/plain, */*'}
+
+
+def update_headers(headers):
+    if headers:
```

Comment:
`if headers is None`? This code seems complicated for what it does. Why isn't just a `headers.update(REQUIRED_HEADERS)` good enough?